data-esnext.js

```diff
@@ -716,21 +716,23 @@ exports.tests = [
   name: 'class decorators',
   category: STAGE2,
   significance: 'medium',
-  spec: 'https://github.com/wycats/javascript-decorators',
+  spec: 'https://tc39.github.io/proposal-decorators/',
   exec: function(){/*
     class A {
       @nonconf
       get B() {}
     }
-    function nonconf(target, name, descriptor) {
-      descriptor.configurable = false;
-      return descriptor;
+    function nonconf(target) {
+      target.descriptor.configurable = false;
+      target.key = 'C';
+      return target;
     }
-    return Object.getOwnPropertyDescriptor(A.prototype, "B").configurable === false;
+    return Object.getOwnPropertyDescriptor(A.prototype, 'B') === undefined &&
+           Object.getOwnPropertyDescriptor(A.prototype, 'C').configurable === false;
   */},
   res: {
-    babel: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Babel 6 still has no official support decorators, but you can use <a href='https://github.com/loganfsmyth/regenerator-plugin-transform-decorators-legacy'>this plugin</a>."},
-    typescript: true,
+    babel: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Decorators are disabled in Babel v6, pending a proposal update – see <a href='https://github.com/babel/babel/issues/2645'>babel/babel#2645</a>."},
```

Comment:
Work is being done, but instead of just throwing an error we are going to move the legacy transform as the one in the package, and yes we will update when landed